src/Listener/ApiQueryLogListener.php

```diff
@@ -106,6 +106,10 @@ protected function _getSources()
      */
     protected function _getSource($source)
     {
-        return ConnectionManager::get($source);
+        try {
+            return ConnectionManager::get($source);
+        } catch (\Cake\Datasource\Exception\MissingDatasourceConfigException $e) {
```

Comment:
This is not the best place to ignore... The best place is where this method is called from